### ISSUE TITLE
Latest canary causes Webpack to panic

### ISSUE THREAD

**Issue by iwatakeshi** [vercel/next.js#17072]
# Bug report

## Describe the bug

Hi, I wanted to try out Webpack 5. However, when I start the dev server, webpack panics with an error message:

```
ValidationError: Invalid configuration object. Webpack has been initialized using a configuration object that does not match the API schema.
 - configuration[0].output.environment should be an object:
   object { arrowFunction?, bigIntLiteral?, const?, destructuring?, dynamicImport?, forOf?, module? }
   -> The abilities of the environment where the webpack generated code should run.
 
 ...
 
errors: [
    {
      keyword: 'type',
      dataPath: '[0].output.environment',
      schemaPath: '#/definitions/Environment/type',
      params: [Object],
      message: 'should be object',
      schema: 'object',
      parentSchema: [Object],
      data: 5
    },
    {
      keyword: 'type',
      dataPath: '[1].output.environment',
      schemaPath: '#/definitions/Environment/type',
      params: [Object],
      message: 'should be object',
      schema: 'object',
      parentSchema: [Object],
      data: 5
    }
  ],
```
## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Create a new project
2. Upgrade next to latest canary
3. Add latest webpack (`5.0.0-beta.30`) to resolutions

Codesandbox:

https://codesandbox.io/s/webpack-5-repro-5vvzs (You may need to restart the server to see the errors)

## Expected behavior

For magic to happen.

## System information

- OS: macOS, Windows
- Browser (if applies): Chrome, Firefox Developer Edition
- Version of Next.js: `^9.5.4-canary.15`
- Version of Node.js: `14.x`

**Comment by jensmeindertsma** [comment]
A fix for this is ready, track #17045.

**Comment by jensmeindertsma** [comment]
The fix has now been merge, I suspect a new canary release will be out soon, so you will be able to use the latest Webpack 5 beta 👍

**Comment by iwatakeshi** [comment]
@jensmeindertsma Awesome. I'll go ahead and close this issue then.

**Comment by ijjk** [comment]
Hi, the above PR has been released in `v9.5.4-canary.16` of Next.js

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
